testing/integration_tests/test_response.json

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
Might want to specify in the file name what test this is for (I'm guessing there are going to be more example responses for different tests in the future)

Reply:
Changed to match test name